utils/params.go

```diff
@@ -95,15 +95,14 @@ type Project struct {
 
 func (p *Project) setDefaultsIfNeeded() error {
```

Comment:
Please add a test to the multiple working dirs. 
Also - adjust the documentation (I guess it's a "," delimiter) - need to point that specifically.